_data/navigation.yml

```diff
@@ -18,3 +18,18 @@ docs:
         url: /docs/getting-started-on-gke/
       - title: "with RBAC"
         url: /docs/running-on-gke-with-rbac/
+  - title: Invokers
+    url: /invokers/
+    children:
+      - title: "Command"
```

Comment:
Can we derive this list from the collection? It's annoying to need to update the nav every time we add a new invoker.

Reply:
yeah, agree, but autocomputing the sidebar nav from multiple collections requires liquid tag gymnastics that i'm a bit reluctant to spend time on - especially considering that this jekyll docs theme is temporary/throwaway - open to suggestions tho if you know how.